pkgs/development/python-modules/tensorflow/bin.nix

```diff
@@ -103,6 +111,11 @@ buildPythonPackage {
 
     pushd dist
 
+    for f in tensorflow-*+nv*.whl; do
+      # e.g. *nv24.07* -> *nv24.7*
+      mv "$f" "$(sed -E 's/(nv[0-9]+)\.0*([0-9]+)/\1.\2/' <<< "$f")"
+    done
+
```

Comment:
why is this logic necessary?

Reply:
The file name from NVIDIA does not exactly match the name in the wheel metadata. `wheel unpack` does not like this inconsistency, and refuses to run.

Reply:
gotcha, let's please make a note of this in a code comment placed outside of the nix string

Reply:
This chunk was previously discussed at https://github.com/NixOS/nixpkgs/pull/334996#discussion_r1726018319. I agree this deserves a better note but also I don't think we should keep blocking this PR, it's been well over a month now